stories/feed-input/feed-input.stories.js

```diff
@@ -0,0 +1,90 @@
+    </div>
+`;
+
+notEmpty.storyName = 'With some text';
+notEmpty.parameters = {
+    docs: {
+        storyDescription: 'Example with a one row text'
+    }
+};
+
+export const inputGrowth = () => `
+    <div class="fd-feed-input" role="region" aria-label="Feed input height growth">
+        <div class="fd-feed-input__thumb" style="background-image: url(http://lorempixel.com/460/620/people/7/)" role="img" title="Thumbnail"></div>
+        <textarea class="fd-textarea fd-feed-input__textarea" id="textarea-3" aria-label="Feed" rows="3" placeholder="Post something here">With every line\ntextarea growths up\nuntil it reaches the max height</textarea>
+        <button class="fd-button sap-icon--feeder-arrow fd-feed-input__submit-button" aria-label="Submit" title="Submit" aria-disabled="false"></button>
+    </div>
+`;
+
+inputGrowth.storyName = 'Multiple lines';
+inputGrowth.parameters = {
+    docs: {
+        storyDescription: 'Every new line extends textarea height until maximum height is reached'
+    }
+};
+
+export const overMaxHeight = () => `
+    <div class="fd-feed-input" role="region" aria-label="Feed input over max height">
+        <div class="fd-feed-input__thumb" style="background-image: url(http://lorempixel.com/460/620/people/7/)" role="img" title="Thumbnail"></div>
+        <textarea class="fd-textarea fd-feed-input__textarea" id="textarea-4" rows="18" aria-label="Feed" placeholder="Post something here">Over max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height</textarea>
+        <button class="fd-button sap-icon--feeder-arrow fd-feed-input__submit-button" aria-label="Submit" title="Submit" aria-disabled="false"></button>
+    </div>
+`;
+
+overMaxHeight.storyName = 'Over max height';
+overMaxHeight.parameters = {
+    docs: {
+        storyDescription: 'Once textarea reaches the maximum height the textarea scroll is shown'
+    }
+};
+
+export const rtl = () => `
+<div dir="rtl">
+    <div class="fd-feed-input" role="region" aria-label="Feed input with RTL support">
+        <div class="fd-feed-input__thumb" style="background-image: url(http://lorempixel.com/460/620/people/7/)" role="img" title="Thumbnail"></div>
+        <textarea class="fd-textarea fd-feed-input__textarea" id="textarea-5" rows="1" aria-label="Feed" placeholder="Post something here">POst message here</textarea>
+        <button class="fd-button sap-icon--feeder-arrow fd-feed-input__submit-button" aria-label="Submit" title="Submit" aria-disabled="false"></button>
+    </div>
+</div>
+`;
+
+rtl.storyName = 'RTL example';
+rtl.parameters = {
+    docs: {
+        storyDescription: 'Right-to-left example'
```

Comment:
"**The right-to-left feed input displays input with text and an active submit button, however it is positioned on the right side of the screen.**"